backend/src/services/secret-v2-bridge/secret-v2-bridge-dal.ts

```diff
@@ -27,34 +31,14 @@ export const SecretDalCacheKeys = {
   getSecretDalVersion: (projectId: string) => {
     return `${SecretDalCacheKeys.productKey}:${projectId}:${TableName.SecretV2}-dal-version`;
   },
-  findByFolderIds: (
+  getSecretsOfServiceLayer: (
```

Comment:
nit: this function name doesn't seem to reflect what it is doing? what about: `getSecretsByPermissionCacheKey`